src/lp/events.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * LP specific events.
+ */
+export enum LiquidityEventName {
+  ADD_LIQUIDITY = 'Add Liquidity',
+  COLLECT_LIQUIDITY = 'Collect Liquidity',
+  MIGRATE_LIQUIDITY = 'Migrate Liquidity',
+  REMOVE_LIQUIDITY = 'Remove Liquidity',
+  SELECT_FEE_POOL = 'Select Fee Pool',
```

Comment:
Where does this selection happen / can we be more specific in the name? What is this actually meant to capture?